_includes/conference.html

```diff
@@ -136,6 +136,17 @@ <h3>Silver</h3>
           </a>
         </div>
       </div>
+      <h3>Bronze</h3>
+      <div class="container">
+        <div class="row">
+          <div class="conference-section col-lg-12 col-md-12">
+            <a href="https://www.metrumrg.com/" target="_blank">
+              <img class="sponsor bronze" src="img/conference/metrymrg-logo.png" alt="Metrum" />
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
This doesn't look like the other sections

```
        <div class="row">
          <h3>Gold</h3>
        </div>
```

followed by a row for each sponsor
```
        <div class="row">
          <a href="https://www.esqlabs.com/" target="_blank">
            <img class="sponsor gold" src="{{ '/img/conference/logo7.png' | relative_url }}" alt="esqLABS" />
          </a>
        </div>
```